fractal-first/architecture/src/main/java/com/yanchware/fractal/pocs/migration/architecture/configuration/Constants.java

```diff
@@ -0,0 +1,8 @@
+package com.yanchware.fractal.pocs.migration.architecture.configuration;
+
+public class Constants {
+  public static final String DEPLOYER_SSH_KEY_SECRET_ID = "fractal-custom-workload-agent-ssh-key";
```

Comment:
Suggestion: This could be a Enum.

Reply:
Does it make sense to have an enum here? I usually use enums when defining a range of interchangeable values within a finite set (like Region, for instance) these are unrelated constants.

Reply:
At least, in Java, an enum is a special data type that enables for a variable to be a set of predefined constants.
And you guarantee that it is a singleton object in the system. :)

Reply:
Still not convinced... let's push this through and you can give me a link of somewhere where it is used so that we can then adopt this at scale 😉